src/main/kotlin/translations/lang/spellcheck/LangKeySplitter.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2020 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.translations.lang.spellcheck
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.spellchecker.inspections.BaseSplitter
+import com.intellij.spellchecker.inspections.IdentifierSplitter
+import com.intellij.util.Consumer
+
+object LangKeySplitter : BaseSplitter() {
+    override fun split(text: String?, range: TextRange, consumer: Consumer<TextRange>?) {
```

Comment:
Same here, `consumer` does not need to be nullable.

Reply:
I have removed this nullability marker.